core/src/test/scala/org/apache/spark/sql/delta/DeltaSourceDeletionVectorsSuite.scala

```diff
@@ -157,15 +158,22 @@ trait DeltaSourceDeletionVectorTests extends StreamTest with DeletionVectorsTest
     val shouldFailAfterCommands = expectations.exists(_.isInstanceOf[ExpectFailure[_]])
 
     val baseActions: Seq[StreamAction] = Seq(
-      StartStream(
-        Trigger.ProcessingTime("10 seconds"),
-        new StreamManualClock(System.currentTimeMillis())),
-      AdvanceManualClock(10L * 1000L),
+      StartStream(Trigger.ProcessingTime(1000), clock),
+      AdvanceManualClock(1000L),
```

Comment:
this part is basically no different from before .. isnt it?

Reply:
Yeah, just got rid of some unnecessary large numbers.

Reply:
Makes it clearer imo that all the units match up as well.

Reply:
But the numbers themselves dont matter since its manual clock.. isnt it? just make sure.

Reply:
But the numbers themselves dont matter since its manual clock.. isnt it? just making sure.